include/PianoRoll.h

```diff
@@ -198,7 +199,8 @@ protected slots:
 		stmaUnmarkAll,
 		stmaMarkCurrentSemiTone,
 		stmaMarkCurrentScale,
-		stmaMarkCurrentChord,
+        stmaMarkCurrentChord,
```

Comment:
Ah ok, that's annoying.  I think I saw that but forgot about it in the
process of setting up my windows build, so I can fix that today.
On Sep 18, 2015 7:37 AM, "Tres Finocchiaro" <notifications@github.com>
wrote:

> In include/PianoRoll.h
> <https://github.com/LMMS/lmms/pull/2351#discussion_r39849927>:
>
> > @@ -198,7 +199,8 @@ protected slots:
> >  		stmaUnmarkAll,
> >  		stmaMarkCurrentSemiTone,
> >  		stmaMarkCurrentScale,
> > -		stmaMarkCurrentChord,
> > +        stmaMarkCurrentChord,
>
> I think the tab formatting is wrong here as well as the following line.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LMMS/lmms/pull/2351/files#r39849927>.
>